client/src/pages/General/AccessibilityPage/index.js

```diff
@@ -0,0 +1,158 @@
+            conformance level ‘AA’; we continually work with stakeholders to
+            ensure that conformance level ‘A’ is adhered to as a minimum. If you
+            experience any accessibility issue on this site or have any comment,
+            please
+            <S.Link to="/"> contact us</S.Link>.
+          </T.P>
+        </Col>
+      </Row>
+      <Row mb="6" mbM="4">
+        <Line />
+      </Row>
+      <Row mb="6" mbM="4">
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.H3 color="black" weight="bold">
+              Adjusting Text Size
+            </T.H3>
+          </Col>
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
+              Internet Explorer
+            </T.P>
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              Go to “View” on the menu bar > Select text size / zoom
+            </T.P>
+          </Col>
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
+              Firefox
+            </T.P>
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              Go to “View” on the menu bar > Select text size / zoom.
+              Alternatively hold down the “Ctrl” button on your keyboard and
+              press the plus (+) key to increase text size. To reduce the latter
+              hold down the “Ctrl” button and press the minus (-) key.
+            </T.P>
+          </Col>
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              Please note that the above settings may differ depending on the
+              browser version.
+            </T.P>
+          </Col>
+        </Row>
+      </Row>
+      <Row mb="6" mbM="4">
+        <Line />
+      </Row>
+      <Row mb="6" mbM="4">
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.H3 color="black" weight="bold">
+              Text To Speech
+            </T.H3>
+          </Col>
+        </Row>
+        <Row mb="6" mbM="4">
+          <T.P color="gray8">
+            Many computers and mobile devices today have built in text-to-speech
+            software. Here are guides for each of the major browsers and
+            devices:
+          </T.P>
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8" weight="bold">
+              Chrome
+            </T.P>
+          </Col>
+          <Col w={[4, 12, 12]}>
+            <T.P color="gray8">
+              <S.A href="https://www.google.fr/">
+                Click here to download Google Speak
+              </S.A>{' '}
+              and select the Add to Chrome button
+            </T.P>
+          </Col>
+        </Row>
+        <Row mb="6" mbM="4">
+          <Col w={[4, 12, 12]}>
```

Comment:
```suggestion
          <Col w={[4, 12, 12]} mb="6" mbM="4">
```
remove unused Row you can add the margin props to the Col component too

Reply:
Done